src/main/java/utils/SwipeHelp.java

```diff
@@ -0,0 +1,19 @@
+package utils;
+
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.MobileElement;
+
+public class SwipeHelp {
```

Comment:
I would change this class' name to SwipeHelper, but take this as a suggestion

Reply:
I agree